src/runtime_src/core/common/ishim.h

```diff
@@ -174,6 +174,11 @@ struct ishim
   virtual void
   register_xclbin(const xrt::xclbin&) const
   { throw not_supported_error{__func__}; }
+
+  //Exec Buf with ctx handle.
+  virtual void
+  exec_buf_ctx(xclBufferHandle /*boh*/, const xrt::hw_context& /*hwctx*/) const
+  { throw not_supported_error{ __func__ }; }
```

Comment:
```suggestion
  exec_buf_ctx(xclBufferHandle boh, const xrt::hw_context& /*hwctx*/) const
  { 
     // Context aware execution is an opt-in.  If not supported, then just call legacy exec_buf
     exec_buf(boh); 
  }
```

Reply:
addressed your comment. Removed const function as it is calling non-cont function.